### ISSUE TITLE
Is it correct? the shape of the input in training.

### ISSUE THREAD

**Issue by rayly2022** [aliseyfi75/COSCI-GAN#2]
I found the input of stock dataset in the training of this model is different from others.
`   ##
    try:
        with open('../Dataset/'+dataset+'.csv', 'rb') as fh:
            df = pd.read_csv(fh)
    except:
        with open('../Dataset/'+dataset+'.pkl', 'rb') as fh:
            df = pickle.load(fh)

    ##
    data = df.sample(frac=real_data_fraction).reset_index(drop=True)

    if 'ID' not in data.columns:
        data['ID'] = np.zeros(len(data))

    ##

    train_id = torch.tensor(data['ID'].values.astype(np.float32))
    train = torch.tensor(data.drop('ID', axis = 1).values.astype(np.float32)) 
    train_tensor = torch.utils.data.TensorDataset(train, train_id) `

I debug the code, and feel puzzled.

**Comment by aliseyfi75** [comment]
Hi!

Thanks for opening this issue. Feel free to open a PR for fixing the problem; I will review it and merge your PR afterward. Thanks for contributing to this project!
